app/views/tradingpremises/where_are_trading_premises.scala.html

```diff
@@ -34,22 +33,20 @@
 
     @form(f, controllers.tradingpremises.routes.WhereAreTradingPremisesController.post(index, edit)) {
 
-        @input(
-            field = f("tradingName"),
-            labelText = "tradingpremises.yourtradingpremises.tradingname"
-        )
-
         @fieldset(
-            legend = "legend.address",
-            legendHidden = true,
+            heading = "tradingpremises.yourtradingpremises.title",
```

Comment:
are we not changing every fieldset to fieldsetWithErrorGroup for consistency?

Reply:
This is one of the off-pattern ones that was briefly mentioned in standup. We don't have a full idea of how to implement for multiple inputs